docs/src/examples/gs2-example_fr.adoc

```diff
@@ -1,14 +1,14 @@
 :lang: fr
 :toc:
 
+[[cha:gs2-spindle]]
+
 = Broche avec variateur GS2
 
 == Exemple
  
 Cet exemple montre les connexions demandées pour utiliser un variateur de 
 fréquence fourni par la société Automation Direct pour piloter une broche.
-footnote:[En Europe on trouve une gamme de produits identiques sous la marque
-Omron.]
```

Comment:
This is a useful addition that may avoid quite a bunch of recurring questions in the forum !
Instead of deleting them I have kept them and only matched line numbers.

Reply:
I underestimated the importance of this footnote, I must admit. But also, I am not really allowed to do semantic changes to the English. Could you please somehow adopt that footnote and prepare a PR to the English?

Reply:
Hm. I have made a few "From _fr:"-like insertions and this could also sneak in this way for an easy scrutiny by Seb et al. I think I just was not really convinced that this was important/informative enough to stand on its own.

Reply:
I added this footnote in english source without structure modification. Will push it with other typo fixes I have collected allong the way...